src/components/transactions/[txid]/TransactionDetails.tsx

```diff
@@ -0,0 +1,117 @@
+import { Transaction, TransactionVin, TransactionVout } from '@defichain/whale-api-client/dist/api/transactions'
+import { NetworkName, useNetworkObject } from '@contexts/NetworkContext'
+import { IoArrowForwardOutline } from 'react-icons/io5'
+import { fromScriptHex } from '@defichain/jellyfish-address'
+import BigNumber from 'bignumber.js'
+
+interface TransactionDetailsProps {
+  transaction: Transaction
+  vins: TransactionVin[]
+  vouts: TransactionVout[]
+  fee: BigNumber
+}
+
+export function TransactionDetails (props: TransactionDetailsProps): JSX.Element {
+  const transaction = props.transaction
+  const vins = props.vins
+  const vouts = props.vouts
+  const networkName = useNetworkObject().name
+  const fee = props.fee
+
+  return (
+    <>
+      <h1 className='font-medium text-2xl mt-6' data-testid='details-subtitle'>Details</h1>
+
+      <div className='mt-5 flex flex-col space-y-6 items-start lg:flex-row lg:space-x-8 lg:space-y-0'>
+        <TransactionDetailsLeft transaction={transaction} vins={vins} vouts={vouts} networkName={networkName} />
+        <div className='flex items-center justify-center text-gray-600 w-full lg:w-auto lg:h-20'>
+          <IoArrowForwardOutline className='transform rotate-90 lg:rotate-0' size={24} />
+        </div>
+        <TransactionDetailsRight transaction={transaction} vins={vins} vouts={vouts} networkName={networkName} />
+      </div>
+
+      <TransactionDetailsSummary transaction={transaction} vins={vins} fee={fee} />
+    </>
+  )
+}
+
+function TransactionDetailsLeft (props: { transaction: Transaction, vins: TransactionVin[], vouts: TransactionVout[], networkName: NetworkName }): JSX.Element {
+  return (
+    <div className='w-full lg:w-1/2 max-h-screen overflow-y-auto pr-1'>
```

Comment:
```suggestion
    <div className='w-full lg:w-1/2 max-h-screen'>
```

don't overflow better UX, let it content scroll naturally